src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -322,6 +321,11 @@ private function loadForm()
     private function setCustomHTML5ErrorMessages(array $item, SpoonFormAttributes $formField)
     {
         foreach ($item['validations'] as $validation) {
+            // we need to change it here since it is saved like this in the database and we can't change that for now.
+            if ($validation['type'] === 'numeric') {
```

Comment:
Well, it should.
Maybe we should also add a `@deprecated`-line, so this code can be removed.